mirrord/layer/src/exec_utils.rs

```diff
@@ -70,12 +70,31 @@ pub(super) fn patch_if_sip(path: &str) -> Detour<String> {
     let skip_patch_binaries = SKIP_PATCH_BINARIES
         .get()
         .expect("skip patch binaries not set");
+    // use args for logging only, so no need to fail if they're not set
+    let args = EXECUTABLE_ARGS
+        .get()
+        .map(|exec_args| exec_args.args.clone())
+        .unwrap_or_default();
```

Comment:
```suggestion
    let args = EXECUTABLE_ARGS
        .get()
        .map(|exec_args| exec_args.args.clone());
```
args are optional in `SipLogInfo`